migrator/database/tests/test_sqlserver.py

```diff
@@ -4,19 +4,19 @@
 
 from pymssql import OperationalError 
 
-from database.SQLServerDatabase import SQLServerDatabase 
-from server.SQLServer import SQLServer
+from migrator.database.SQLServerDatabase import SQLServerDatabase 
+from migrator.server.SQLServer import SQLServer
 
 
 logging.basicConfig(
-    filename='TestDatabase_SQLServer.log',
+    filename='TestServer_SQLServer.log',
```

Comment:
Filename here should not change. Should be reverted to `TestDatabase_SQLServer.log`.